releasenotes/notes/added-runtime-params-class-a458e64bea4ec8f6.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Added option to execute ``providers.run`` with ProvidersRequestParams, rather than simply a Dict object, which for auto-complete features to be used.
```

Comment:
Any mentions of methods and classes should have documentation links. Check out the other release notes for reference.

Reply:
Fixed!